tests/plugins/flash/manual/error.md

```diff
@@ -0,0 +1,16 @@
+@bender-tags: 4.17.0, feature, 4866
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, flash
```

Comment:
It would be good to add some minimal editor configuration like basicstyles, link, and undo, so the test doesn't look broken (without toolbar buttons).